DependencyInjection/Configuration.php

```diff
@@ -204,6 +218,12 @@ function($v) {
                     $c['report_only'] = false;
                     return $c;
                 })
+            ->end()
+            ->validate()
+                ->ifTrue(function($v) {
+                    return !in_array($v['hash']['algorithm'], array('sha256', 'sha384', 'sha512'), true);
```

Comment:
This is useless as the configuration uses an enumNode which already does exactly this validation